robert_beverage/scripts/web_server

```diff
@@ -1,23 +1,33 @@
 #!/usr/bin/env python
+import os
 import rospy
 import socket
 import SocketServer
 import SimpleHTTPServer
+from urlparse import urlparse
+import rospkg
 
 rospy.init_node('web_server_node')
 rospy.loginfo('Inside web_server script')
+rospack = rospkg.RosPack()
 
 HOST = socket.gethostname()
 PORT = 8080
 address = ("", PORT)
+os.chdir('/')
```

Comment:
you are allowing to access root folder by default. Which is not good from security perspective on web servers. Please switch it to package folder.